### PR TITLE
Add pointer to Discord Server

### LINKED ISSUE #3083: Add link to our Discord Server
We now have a Discord server as an additional communication channel for interactive discussions as an alternative to Gitter. We should at least mention it in the README and maybe later (?) also on the website.

### PR DESCRIPTION
**Pull request by FlorianHockmann**
This justs adds Discord as an alternative to Gitter. We can later remove Gitter if we think that user / contributor experience is better on Discord.

Fixes #3083

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,9 +56,11 @@ tools:
...
 ## Community
 
-* GitHub Discussions: see [`GitHub Discussions`](https://github.com/JanusGraph/janusgraph/discussions) 
+* GitHub Discussions: see [`GitHub Discussions`](https://github.com/JanusGraph/janusgraph/discussions)
   for all general discussions and questions about JanusGraph
-  
+
+* Discord for interactive discussions and questions about JanusGraph: [Join the server](https://discord.gg/5n4fjv4QAf)
+
 * Chat rooms on Gitter:
```
Comment (cdegroc) [inline README.md:64]: Should Gitter be removed now that Discord is up?

Comment (FlorianHockmann) [reply README.md:64]: That's a good question. @porunov [suggested on janusgraph-dev](https://lists.lfaidata.foundation/g/janusgraph-dev/message/1575) that we keep Gitter for now as we have 864 members there and wait until Discord has a similar popularity before we abandon it.
Now we could of course keep Gitter active but stop promoting it in the README & the issue template (I just noticed that I forgot to add Discord there as well, I'll add that).
I personally think we should wait a bit to see whether users and contributors actually want to use Discord and whether the experience is better on Discord vs Gitter (e.g., are questions from users actually answered) before we completely remove all mentions of Gitter.

Comment (FlorianHockmann) [reply README.md:64]: Updated the commit & PR description.

**Review by porunov — APPROVED** [review]
LGTM. Thank you @FlorianHockmann !

**Pull request merged**